### ISSUE TITLE
[JoSS] Ayaan's Review Thread

### ISSUE THREAD

**Issue by ayaanhossain** [milnus/Magphi#10]
Hey guys, I'll be reviewing this repository and the paper on behalf of JoSS! This is my main review thread, in which I'll discuss things with you guys. If there is a specific issue we need to address, I'll open a separate issue and link it here. Once my review is done, and everything is resolved, I'll close this issue.

**Comment by milnus** [comment]
Other reviewers were found on a later date
